tools/harness-thci/OpenThread.py

```diff
@@ -2813,7 +2813,7 @@ def MGMT_ACTIVE_SET(
                     rotationTime = listSecurityPolicy[0]
                     # bit order
                     if TESTHARNESS_VERSION == TESTHARNESS_1_2:
-                        if self.DeviceCapability != DevCapb.V1_1:
+                        if self.DeviceCapability != DevCapb.V1_1 and len(listSecurityPolicy) > 2:
```

Comment:
It seems to me that the whole block of code can be simplified from:

```
if TESTHARNESS_VERSION == TESTHARNESS_1_2:
    if self.DeviceCapability != DevCapb.V1_1 and len(listSecurityPolicy) > 2:
        policyBits = listSecurityPolicy[2] << 8 | listSecurityPolicy[1]
    else:
        policyBits = listSecurityPolicy[1]
else:
    policyBits = listSecurityPolicy[1]
```

To:
```
if len(listSecurityPolicy) > 2:
    policyBits = listSecurityPolicy[2] << 8 | listSecurityPolicy[1]
else:
    policyBits = listSecurityPolicy[1]
```
Since the test script already discerns which is the Thread version of the device.

Reply:
Thanks, I have checked the proposed change and it works, I updated the code.